package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "leaderboard",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
- [ ] The `Scripts`  section in your package.json file is missing a build script to run your project. For this reason, `npm start or dev` is failing. Kindly add it up 
- You can do well to revisit your webpack exercise, to have a reference guide 👍 

### Sample code:
```
  "scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack --config webpack.config.js --watch"
  }
```